.github/renovate.json5

```diff
@@ -0,0 +1,52 @@
+{
+    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+    "extends": [
+        "github>cjorge-graphops/renovate-config"
```

Comment:
@cjorge-graphops is this what we want here?

Reply:
where is this extended config stored at?

Reply:
totally isn't, well spotted :bow: